src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -37,6 +37,34 @@ void RotatedSPOs::setRotationParameters(const std::vector<RealType>& param_list)
   params_supplied = true;
 }
 
+// Construct a list of the matrix indices for non-zero rotation parameters
+//  (The structure for a sparse representation of the matrix)
+// Only core->active rotations are created.
+void RotatedSPOs::createRotationIndices(int nel, int nmo, RotationIndices& rot_indices)
```

Comment:
It think the comments are better with the function declaration. If they are better for the function definition, comment lines can be better with the actual code.

Reply:
Yes, this is our standard practice.